stdlib/hashtbl.mli

```diff
@@ -249,6 +249,20 @@ val of_seq : ('a * 'b) Seq.t -> ('a, 'b) t
     will appear in the table.
     @since 4.07 *)
 
+val of_list : ('a * 'b) list -> ('a, 'b) t
+(** [Hashtbl.of_list l] is a table built from the given bindings. The bindings
+    are added in the same order they appear in [l], using {!add}.
+
+    @since 4.09 *)
+
+val to_list : ('a, 'b) t -> ('a * 'b) list
+(** [Hashtbl.to_list h] is the list of bindings in [h]. The order in which the
+    bindings appear in the list is unspecified. However, if the table contains
+    several bindings for the same key, they appear in order of introduction,
+    that is, the most recent binding appears last.
```

Comment:
Might be worth adding a test to check this property.

Reply:
Done.